{
  "contest": 367,
  "handle": "0xpetern",
  "risk": "3",
  "title": "Possible Reentrancy issue with the withdraw function",
  "issueId": 1201,
  "issueUrl": "https://github.com/code-423n4/2024-04-dyad-findings/issues/1201"
}